src/include/miopen/solver.hpp

```diff
@@ -2450,6 +2450,79 @@ struct ConvAsmBwdWrW3x3 final : ConvTunableSolver<PerformanceConfigAsmDirect3x3W
                              const PerformanceConfigAsmDirect3x3WrW& config) const override;
 };
 
+struct PerformanceConfigConvWinoFuryRxS : PerfConfigBase<PerformanceConfigConvWinoFuryRxS>
+{
+    int n_groups;
+    PerformanceConfigConvWinoFuryRxS(int n_groups_);
+    PerformanceConfigConvWinoFuryRxS() : PerformanceConfigConvWinoFuryRxS(-1) {}
+    PerformanceConfigConvWinoFuryRxS(bool) : PerformanceConfigConvWinoFuryRxS(1) {}
+
+    template <class Self, class F>
+    static void Visit(Self&& self, F f)
+    {
+        f(self.n_groups, "n_groups");
+    }
+    int GetNGroups() const { return n_groups; }
+
+    void HeuristicInit(const ConvolutionContext&, const ProblemDescription&, size_t, size_t);
+    bool IsValidValue() const;
+    bool SetNextValue(const ProblemDescription&);
+    bool IsValid(const ConvolutionContext& ctx, const ProblemDescription&) const
+    {
+        return IsValid(ctx);
+    }
+    bool IsValid(const ConvolutionContext&) const;
+    bool operator==(const PerformanceConfigConvWinoFuryRxS& other) const;
+};
+
+template <size_t Winodata, size_t Winofilter>
+struct ConvWinoFuryRxS final : ConvTunableSolver<PerformanceConfigConvWinoFuryRxS>
+{
+    const std::string& SolverDbId() const override { return GetSolverDbId(); }
+
+    static const std::string& GetSolverDbId()
+    {
+        static const std::string dbId = std::string("ConvWinoFuryRxSf")
+                                            .append(std::to_string(Winodata))
+                                            .append("x")
+                                            .append(std::to_string(Winofilter));
+        return dbId;
+    }
```

Comment:
Renaming is not required.

```suggestion
    const std::string& SolverDbId() const override
    {
        return GetSolverDbId<ConvWinoFuryRxS<Winodata, Winofilter>>();
    }
```